setup_instructons/readme.md

```diff
@@ -6,22 +6,29 @@ Virtualization can get complicated. There are many options for VM technology and
 
 We've marked the solution we recommend for each scenario. These are the options that we feel are most reliable and easiest for us to help you with should you have any problems.
 
-## For Windows Computers
+## ROS2 and IsaacROS
+The teams with access to a Jetson platform are using ISAAC ROS built and maintained by NVIDIA. ISAAC ROS is built on ROS 2 Humble and has all the capabilites of ROS 2 Humble and can nominally be used as such. ISAAC ROS comes with a few libraries that can prove to be useful but can only be run optimized on Nvidia hardware (use the isaac_ros development container) [instructions] (isaac_ros_WSL.md)
+
+## Recomeded Method For ALL Operating Systems:
+- [Docker](docker.md)
 
+
+## For Windows Computers
+If you have an NVIDIA GPU we recomend using WSL for best use of the CUDA acceration
 - [Windows Subsystem for Linux](wsl.md) 
-- [VirtualBox](virtualbox.md)
+~~ - [VirtualBox](virtualbox.md)~~
```

Comment:
Is the intent here to strikethrough these links? If so, you can't have spaces right next to the tildes. And striking through bullets for lists can cause weird rendering. Usually better to just strikethrough the text. (Or just delete these sections entirely if they're no longer recommended.)

```suggestion
- ~~[VirtualBox](virtualbox.md)~~
```